package.json

```diff
@@ -52,15 +52,16 @@
     "documentdb": "^1.12.2",
     "durable-functions": "^1.2.2",
     "express": "^4.15.3",
-    "fp-ts": "1.12.0",
+    "fp-ts": "1.17.4",
     "html-to-text": "^4.0.0",
-    "io-functions-commons": "^7.2.0",
+    "io-functions-commons": "7.2.0",
```

Comment:
we will upgrade to 7.3.0 in another pr as it adds support for prescription metadata into messages